lib/modules/datasource/go/releases-goproxy.ts

```diff
@@ -240,6 +257,28 @@ export class GoProxyDatasource extends Datasource {
     return result;
   }
 
+  async getVersionsWithInfo(
+    baseUrl: string,
+    packageName: string
+  ): Promise<Release[]> {
+    const releasesIndex = await this.listVersions(baseUrl, packageName);
+    const releases = await p.map(releasesIndex, async (versionInfo) => {
```

Comment:
i think we need to limit concurrency here. otherwise we'll hammering the API too much.

@rarkins WDYT?

Reply:
Isn't this just copied from deleted line 56? I think if it's Google's GOPROXY then it can probably handle load, we'd be more likely to swamp *ourselves*

Reply:
> Isn't this just copied from deleted line 56?

Yes